commit dc3e86c36b61535c9bc3db5c9ede37cfa415ec9a
Author: Aws Ismail <aws.ismail@windriver.com>
Date:   Thu May 31 18:44:10 2012 -0400

    Added fix for AC_REPLACE_FNMATCH in configure.ac
    
    On a x86-64 target, AC_REPLACE_FNMATCH lead to the following error
    duing the configure step:
    
    | config.status: linking ././fnmatch_.h to ./fnmatch.h
    | config.status: error: ././fnmatch_.h: file not found
    | ERROR: oe_runconf failed
    
    AC_REPLACE_FNMATCH is now deprecated as of AC 2.61. It is replaced
    with AC_FUNC_FNMATCH for the time being. This patch is an update to
    the original patch from the legacy version of corosync in WRL4.3
    
    Upstream Status: Pending
    
    Signed-off-by: Aws Ismail <aws.ismail@windriver.com>

diff --git a/configure.ac b/configure.ac
index 550dcb4..d78c549 100644
--- a/configure.ac
+++ b/configure.ac
@@ -100,7 +100,7 @@ AC_C_VOLATILE
 # Checks for library functions.
 AC_FUNC_CLOSEDIR_VOID
 AC_FUNC_ERROR_AT_LINE
-AC_REPLACE_FNMATCH
+AC_FUNC_FNMATCH
 AC_FUNC_FORK
 AC_PROG_GCC_TRADITIONAL
 AC_FUNC_MALLOC
